packages/compendium-cdk/lib/schema.graphql

```diff
@@ -3,6 +3,8 @@ type Node {
   name: String!
   metadata: NodeMetadata
   version: String!
+  typeId: String!
```

Comment:
@zatkoa do we need to expose typeId directly? (just asking if I missed the usage or it's just for fetching simplicity)

Reply:
I thought that it was necessary for fetching node type (see https://github.com/compendium-app/compendium/pull/3/files#diff-4d6366a27b5f449da4d75d24f413db3bdb160245feb4bd5d0785038cd2da68a4R245), i'm still not quite sure how it works. if the `source` object contains all previously returned data by the parent resolver, i guess it's not needed.

Reply:
It should be the resolver result, but TBH I'm not 100% sure as well :-D (but if I remember correctly from other projects, it should be this way). On the other hand this way we can fetch Type ID for all fetch nodes without needing to do another DynamoDB GetItem for each n.ode

Reply:
Yeah, that's a good point. Speaking of that, i was wondering if it's a good solution to call `getItem` for every node (as we know, that there can be a lot of nodes). What's your opinion on that? It is better to duplicate  `typeName` to each node (to not run an additional queries)? Because that would make updates of the node type more complicated.
The other solution is to get all node types on FE and join them by ID with `node.typeId`.

Reply:
I prefer fetching it than duplicating the data. It's not as efficient, but in case anybody want to update the Node name, we need to create procedure which goes through all the nodes and updates them as well